### PR TITLE
lxhentai & MojoPortalComic tlc

### LINKED ISSUE #2361: [Site Request] hentailx
Name of the website:

Hentailx

• Site: http://lxhentai.com/
• Manga List:
• Manga example: http://lxhentai.com/story/view.php?id=1482&token=MC4yNjA5ODQ0ODQ4Nzg4Mjg1MC4yODQyMjg2MzAzNzAxMTYyNQvv
• Chapter online viewer example: http://lxhentai.com/story/chapter.php?id=6702&token=MC43OTQ1MDM1MjQ3MjczMjgyMC41OTEwODg2NDQ5MzMxNjk2

Languages: Vietnamese

### PR DESCRIPTION
**Pull request by Robonau**
resolves #2361

### CODE REVIEW

File: src/web/mjs/connectors/templates/MojoPortalComic.mjs [hunk 1]
```diff
@@ -10,6 +10,14 @@ export default class MojoPortalComic extends Connector {
         super.label = undefined;
         this.tags = [];
         this.url = undefined;
+
+        this.queryMangaTitle = 'meta[property="og:title"]';
+        this.queryChapter = 'div.list-chapter ul li.row div.chapter a';
+        this.queryPages = 'div.reading div.page-chapter source';
+        this.path = '/?page=';
+        this.queryMangasPageCount = 'div.pagination-outter ul.pagination li:last-of-type a'
+        this.pathMatch = /(\d+)+$/;
```
Comment (27136773) [inline src/web/mjs/connectors/templates/MojoPortalComic.mjs]: Can be simplified to `/(\d+)$/`

File: src/web/mjs/connectors/templates/MojoPortalComic.mjs [hunk 2]
```diff
@@ -34,17 +42,17 @@ export default class MojoPortalComic extends Connector {
...
     }
 
     async _getMangas() {
         let mangaList = [];
-        let request = new Request(this.url + '/?page=', this.requestOptions);
-        let data = await this.fetchDOM(request, 'div.pagination-outter ul.pagination li:last-of-type a');
-        let pageCount = parseInt(data[0] ? data[0].href.match(/\d+$/)[0] : 1);
+        let request = new Request(this.url + this.path, this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryMangasPageCount);
+        let pageCount = parseInt(data[0].href.match(this.pathMatch)[1]);
```
Comment (27136773) [inline src/web/mjs/connectors/templates/MojoPortalComic.mjs]: Previous code handled the use case when no pagination element was found `data[0] ? ... : 1` is it safe to remove this check?

File: src/web/mjs/connectors/LxHentai.mjs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import MojoPortalComic from './templates/MojoPortalComic.mjs';
+
+export default class LxHentai extends MojoPortalComic {
+
+    constructor() {
+        super();
+        super.id = 'lxhentai';
+        super.label = 'LxHentai';
+        this.tags = [ 'manga', 'hentai', 'vietnamese' ];
+        this.url = 'http://lxhentai.com';
```
Comment (27136773) [inline src/web/mjs/connectors/LxHentai.mjs]: May prefer `https`

File: src/web/mjs/connectors/templates/MojoPortalComic.mjs (hunk 2 above)
Comment (Robonau) [reply src/web/mjs/connectors/templates/MojoPortalComic.mjs]: all the connectors that use _getMangas from the template are still are functional

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**